hr_timesheet_name_customer/README.rst

```diff
@@ -0,0 +1 @@
+
```

Comment:
please delete this file or generate readme from https://github.com/OCA/maintainer-tools#readme-generator

Reply:
@Sudarshan1607 could you please check once again? Would be great to have it finally merged

Reply:
> please delete this file or generate readme from https://github.com/OCA/maintainer-tools#readme-generator

@Saran440 could you please review its once again? Would be great to have it merged